index.php

```diff
@@ -111,3 +118,23 @@ function formatCustomers($bookingInfo)
     }
     return $customers;
 }
+
+function formatComponents($bookingInfo)
+{
+    $aux_array = ['component_id', 'product_id', 'date_id', 'date_type', 'product_code', 'start_date', 'end_date', 'rate_breakdown', 'sale_price', 'sale_currency'];
```

Comment:
please try to use a better naming for vars .. this could be something like valid fields array or something more descriptive

Reply:
fixed